tcms/testruns/tests/test_permissions.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name, too-many-ancestors
+
+
+from django.urls import reverse
+from django.utils.translation import ugettext_lazy as _
+
+from tcms.tests import PermissionsTestCase
+from tcms.tests import factories
+
+
+class EditTestRunViewTestCase(PermissionsTestCase):
+    """Test edit view method"""
```

Comment:
If you are adding a doc string make sure it is correct. EditTestRunView is a class, not a method :-)